app/controllers/webhooks.js

```diff
@@ -188,6 +220,14 @@ module.exports = (app) => {
           DonationId: donation.id,
           amount: stripeSubscription.amount / 100,
           currency: stripeSubscription.currency,
+          fxCurrency: balanceTransaction.currency,
+          fxAmount: balanceTransaction.amount,
+          fxRate: donation.amount/balanceTransaction.amount,
+          fxHostFee: parseInt(balanceTransaction.amount*0.05, 10), // TODO: find a better way than hardcoding
+          fxPlatformFee: fees.applicationFee,
+          fxPaymentProcessorFee: fees.stripeFee,
```

Comment:
that logic is already in the `donations` controller. We should put that in the function and unit test it